man/org.freedesktop.login1.xml

```diff
@@ -114,7 +114,7 @@ node /org/freedesktop/login1 {
       LockSessions();
       UnlockSessions();
       KillSession(in  s session_id,
-                  in  s who,
+                  in  s whom,
```

Comment:
is dbus happy with these changes to existing methods?

Reply:
The dbus names are not used in the protocol. So I think this change should be invisible, except for docs. But we probably should wait until after release, in case there are some regressions. E.g. I'm not sure if some high-level libraries doesn't use our introspection data for something.

Reply:
Yeah it's not the protocol, but the introspection and code generation that I am a little worried about

Reply:
We changed the introspection data in the past and there didn't seem to be any problems. In particular, we didn't have parameter names before 09f87228011cb763c3c0a94193d3f4eb0c6cc23b at all and I don't recall any complaints when this changed. This wasn't so long ago.